CMakeLists.txt

```
@@ -0,0 +1,33 @@
+#[[
+This source file is part of the Swift Atomics Open Source Project
+
+Copyright (c) 2020 Apple Inc. and the Swift Numerics project authors
```

For consistency, I believe this should read:

```suggestion
Copyright (c) 2021 Apple Inc. and the Swift project authors
```